frontends/web/src/api/exchanges.ts

```diff
@@ -44,6 +44,7 @@ export type ExchangeDeal = {
 export type ExchangeDeals = {
   exchangeName: string;
   deals: ExchangeDeal[];
+  supported?: boolean
```

Comment:
@thisconnect you preferred this field to be moved to a separate type, right?

Reply:
yes, I think it makes sense that this type (actually all types in `api/*`) describes the data that is coming from the backend. Reading as is, it looks like the backend optionally sends supported boolean which it never sends.

So if `supported ` is added by the frontend, the type should be extended where this transformation happens.

Reply:
makes sense to me, latest change looks good to me, ping @Beerosagos